src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -12,8 +11,7 @@ import {
 } from "@stencil/core";
 import { ICON_TYPES, TEXT } from "./resources";
 import sharedListMethods from "./shared-list-logic";
-import { VNode } from "@stencil/core/dist/declarations";
-import CalciteScrim from "../utils/CalciteScrim";
+import List from "./shared-list-render";
```

Comment:
Nitpick: Is there a more discoverable location for the common list modules?

Reply:
We could put them in utils maybe? Though I think it'd be about time to start adding sub-folders to utils, which might introduce some file reference refactors in other files.
Maybe `src/components/utils/functionalComponents`

Very open to other suggestions